Diary/src/main/java/com/microsoft/device/display/samples/diary/MainApp.kt

```diff
@@ -0,0 +1,57 @@
+package com.microsoft.device.display.samples.diary
+
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.getValue
+import androidx.compose.runtime.mutableStateOf
+import androidx.compose.runtime.saveable.rememberSaveable
+import androidx.compose.runtime.setValue
+import androidx.compose.ui.platform.LocalContext
+import com.microsoft.device.display.samples.diary.ui.pages.CalendarPage
+import com.microsoft.device.display.samples.diary.ui.pages.DiaryPage
+import com.microsoft.device.dualscreen.twopanelayout.TwoPaneLayout
+import java.io.File
+import java.time.LocalDate
+
+/**
+ *  For handling window navigation for Surface Duo, this acts as a main Controller for both the Composable
+ */
+@Composable
+fun MainApp() {
+    val context = LocalContext.current
+    val rootDataDir: File = context.applicationContext.dataDir
+
+    var text by rememberSaveable { mutableStateOf("") }
+    var currentSelectedDate by rememberSaveable {
+        mutableStateOf(LocalDate.now())
+    }
+    var content by rememberSaveable { mutableStateOf("") }
```

Comment:
I would recommend pulling this out and putting it into a model, as well as changing the name of the text to something descriptive so that we know what the text is used for.